src/System.Linq.Expressions/src/System/Dynamic/UpdateDelegates.Generated.tt

```diff
@@ -0,0 +1,332 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ output extension=".cs" #>
```

Comment:
Would a clearer convention perhaps be if this was called "UpdateDelegates.tt" and had `<#@ output extension="Generated.cs" #>`? The current name suggests that this file itself is generated.